src/optimizer/query_graph.c

```diff
@@ -2975,7 +2975,7 @@ get_term_subqueries (QO_ENV * env, QO_TERM * term)
   PT_NODE *pt_expr, *next;
   WALK_INFO info;
 
-  if (QO_IS_FAKE_TERM (term))
+  if (QO_IS_DEP_TERM (term))
```

Comment:
It looks better change the clause "This is a pseudo-term ..." to "This is a dependent-term ..."

Reply:
'pseudo-term introduced to keep track of derived' means 'dependent-term'. so I don't think it needs to be changed.